lib/charliecloud.py

```diff
@@ -259,7 +259,11 @@ def get(self):
          except KeyError:
             # Finally, prompt the user.
             # FIXME: This hangs in Bats despite sys.stdin.isatty() == True.
-            username = input("\nUsername: ")
+            try:
+               username = input("\nUsername: ")
+            except KeyboardInterrupt:
+               INFO('')
```

Comment:
Is this because without the blank message, the error message is on the same line as the prompt? I think you can remove it, because (1) it's very common for control-C to generate weirdly aligned error messages and (2) the mixed level (INFO vs. FATAL) is jarring and seems brittle.